tests/test_parser.py

```diff
@@ -40,6 +54,21 @@ def encoded_dataset(filename):
         return (encoder.content_type, encoder.to_string())
 
 
+class TestingValueTarget(ValueTarget):
+    def __init__(self):
+        super().__init__()
+        self.started = False
+        self.finished = False
```

Comment:
Can this also be moved to the `Target` interface?

Reply:
commented above